docs/index.md

```diff
@@ -16,115 +22,237 @@
+The t-SNE embedding translated the data and found local clusters previously unseen; one of the clades (A1b/131K) was split into three clusters that, when looking at NextStrain’s updated Ha phylogeny, slices the clade in three clean pieces. Looking at the nextstrain clade, this “slicing” in the dimensionality reduction revealed patterns in the tree that were not seen, and this foreshadows Cartography’s possible competency with pinpointing future clade break points.
+
+
+UMAP: UMAP reduced the pairwise distance matrix (Figure 1).
+The UMAP pairwise vs euclidean distance plot gave a pearson coefficient of .471.
+The data points did splay out from the LOESS line, but far less so compared to t-SNE - the more distinguishing characteristics were the clusters shown in the scatter plot itself, revealing UMAP’s approach to visualizing local and global patterns equally (Figure 2).
+The ratio between the medians given by the within vs between violin plot for UMAP was 9:1, revealing tightly clustered data.
+The data points from different clades were far apart and incredibly densely packed.
+The density of the within clade violin plot is at much lower euclidean distances than the density of the between clade violin plot, revealing that it is extremely easy to discern the between vs within clade measure given a euclidean distance (corroborating the observation of UMAP’s densely packed and compartmentalized embedding of these data points) (Figure 3).
+The embedding did translate the data differently than the rendering of the tree, giving much more euclidean space between points that were more genetically diverse and packing genetically similar strains very close together, if not stacked.
+This pattern is clearly shown in the pairwise vs euclidean graph for UMAP, as the LOESS line begins to plateau as the genetic distance gets larger and larger, revealing that UMAP sets a threshold for genetic diversity and keeps points above that threshold very far apart in euclidean space.
+This is different from other embeddings, because PCA, MDS, and t-SNE do not set a threshold for genetic diversity, and instead scale the data based on the largest genetic distance disparity (this makes UMAP’s embedding so densely clustered).
+
+
+## Figure One
+
+<iframe src="https://blab.github.io/cartography/FullLinkedChartClickableFlu.html" style="width: 1200px; height: 400px;" frameBorder="0"></iframe>
+
+## Figure Two
+
+![](FullScatterplotFlu.png)
+
+## Figure Three
+![](FullViolinPlotFlu.png)
+
+
+
+## Zika:
+
+Zika: Zika in this project is used as a test case. 
+As zika genes all exist on the same genome and zika’s genome is considerably longer than H3N2 Influenza genome, the genetic diversity of zika, while low enough to create phylogenetic transmission tree, clades are hard to distinguish from each other - zika transmission trees are often colored by region and country for this reason. 
```

Comment:
How long is the Zika genomes specifically?

While flu is a globally distributed virus that has caused seasonal infections for decades (if not hundreds of years), Zika is a relatively new human pathogenic virus that has a restricted geographic distribution that recapitulates the patterns of viral transmission. For these reasons, Zika trees are usually colored by geographic attributes instead of clades.

Having thought about this a bit more, maybe we should revisit performing this analysis with "clades" and use geographic regions here instead...we can talk about this more offline.